app/src/main/java/com/zulip/android/ZLog.java

```diff
@@ -1,12 +1,14 @@
 package com.zulip.android;
 
+import android.util.Log;
+
 import com.crashlytics.android.Crashlytics;
 
 public class ZLog {
 
     public static void logException(Throwable e) {
-        if (BuildHelper.shouldLogToCrashlytics()) {
-            e.printStackTrace();
+        if (!BuildHelper.shouldLogToCrashlytics()) {
+            Log.e("Error", "oops", e);
```

Comment:
I understand the bug fix in this file, but what's the story with the rest of the commit?